pkg/virt-operator/kubevirt_test.go

```diff
@@ -1604,10 +1680,43 @@ var _ = Describe("KubeVirt Operator", func() {
 			kubecontroller.SetLatestApiVersionAnnotation(kv)
 			addKubeVirt(kv)
 			addInstallStrategy(defaultConfig)
-			addAll(defaultConfig)
+			addAll(defaultConfig, kv)
 			addPodsAndPodDisruptionBudgets(defaultConfig)
 			makeApiAndControllerReady()
 			makeHandlerReady()
+
+			kv.Status.Generations = append(kv.Status.Generations, operatorsv1.GenerationStatus{
+				Group:          "admissionregistration.k8s.io",
+				Resource:       "validatingwebhookconfigurations",
+				Namespace:      "",
+				Name:           "virt-operator-validator",
+				LastGeneration: 0,
+			})
+
+			kv.Status.Generations = append(kv.Status.Generations, operatorsv1.GenerationStatus{
+				Group:          "admissionregistration.k8s.io",
+				Resource:       "validatingwebhookconfigurations",
+				Namespace:      "",
+				Name:           "virt-api-validator",
+				LastGeneration: 0,
+			})
+
+			kv.Status.Generations = append(kv.Status.Generations, operatorsv1.GenerationStatus{
+				Group:          "admissionregistration.k8s.io",
+				Resource:       "mutatingwebhookconfigurations",
+				Namespace:      "",
+				Name:           "virt-operator-mutator",
+				LastGeneration: 0,
+			})
+
+			kv.Status.Generations = append(kv.Status.Generations, operatorsv1.GenerationStatus{
+				Group:          "admissionregistration.k8s.io",
+				Resource:       "mutatingwebhookconfigurations",
+				Namespace:      "",
+				Name:           "virt-api-mutator",
+				LastGeneration: 0,
+			})
+
```

Comment:
i see that you're adding to the kv.Status.Generations list in the normal add functions. Like `addDaemonSet()` for example.

I'm curious why you explicitly set the generations for these webhooks here when I'd think they'd already be added in the `addValidatingWebhook()` shared function.